app/assets/stylesheets/views/_organisations.scss

```diff
@@ -1,24 +1,50 @@
 .organisations {
-  .organisations__section {
+  .organisations-list__section {
     margin-bottom: $gutter * 2;
 
     .department-count {
       @include bold-80;
     }
 
-    .organisations__logo-wrapper {
+    .organisations-list__item {
```

Comment:
Since you're using BEM, we probably don't need to nest all this SASS, would make it a bit more readable.